drivers/usbdev/usbmsc_scsi.c

```diff
@@ -173,6 +173,8 @@ static int    usbmsc_cmdstatusstate(FAR struct usbmsc_dev_s *priv);
  * Private Data
  ****************************************************************************/
 
+static const char *g_productrevision = "0101";
```

Comment:
So this is hardcoded? Why?

Reply:
I don't know about the details of this SCSI field, but USBMSC does not affected when this field is any values. So I wrote it in hard coded anyway.